solution/ui/regulations/eregs-vite/src/components/pagination/PageNumber.vue

```diff
@@ -66,24 +66,3 @@ export default {
     },
 };
 </script>
-
-<style lang="scss" scoped>
-li.page-number-li {
```

Comment:
An example of CSS-in-JS styles that have been moved from here to the `_pagination.scss` stylesheet partial above